packages/dds/tree/src/test/domains/json/jsonGeneratorUtils.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * This file contains a series of utility functions intended to assist with generating random data.
+ */
+
+// returns a decimal number between inclusive of the min and max provided
+export function getRandomNumber(min: number, max: number) {
+    return Math.random() * (max - min) + min;
+}
```

Comment:
Like Canada, you'll want to switch to the PRNG in 'stochastic-test-utils' and use a static seed so that the benchmark measures the same JSON from run to run.

As a bonus, stochastic-test-utils provides helpers to choose integers in a range, choose bools with a given probability, etc.

Reply:
Sounds good. I've swapped it to utilize the PRNG in 'stochastic-test-utils' as of my second commit.

Reply:
...and you should consider adding additional utilities you find lacking, such as Markov Chain generation.

Reply:
Markov has been added as of the 3rd commit (hash 133796)